Readme.md

```diff
@@ -129,6 +129,15 @@ If all of your APIs pass the tests in this repo, please refer to our public docu
 
 [Using On-Premises Data in Place with Snowflake](https://www.snowflake.com/blog/external-tables-on-prem/)
 
+## Troubleshooting
+We expect the storage vendors provide S3-compatible APIs, which should work like S3 APIs. We have observed there are still differences between storage vendors.
+Below are some troubleshooting cases. When your tests fail, please refer to the source code to see what are the test cases.
+1. getBucketLocation tests fail
+   We call getBucketLocation() to retrieve the region for a bucket, because the region is required for SigV4. If you confirm that your service ignore the bucket location (region) in the SigV4 from the request, then you can ignore the test failures for getBucketLocation.
+2. negative tests fail
+   For some negative test cases, AWS S3 returns 404 or 403, while your APIs return 400 or other error code, so your tests fails as the error code is different from what the tests expect.
+   It should be fine if your APIs return reasonable error codes and messages for those negative cases.
```

Comment:
This should be fine as long as your APIs return ...

Reply:
Ack!